src/rrules/low_level_maths.jl

```diff
@@ -95,6 +95,35 @@ function rrule!!(::CoDual{typeof(Base.eps)}, x::CoDual{P}) where {P<:IEEEFloat}
     return zero_fcodual(y), eps_pb!!
 end
 
+@is_primitive MinimalCtx Tuple{typeof(hypot),P,P} where {P<:IEEEFloat}
+function frule!!(::Dual{typeof(hypot)}, x::Dual{P}, y::Dual{P}) where {P<:IEEEFloat}
+    h = hypot(primal(x), primal(y))
+    dh = (primal(x) * tangent(x) + primal(y) * tangent(y)) / h
+    return Dual(h, dh)
+end
+function rrule!!(::CoDual{typeof(hypot)}, x::CoDual{P}, y::CoDual{P}) where {P<:IEEEFloat}
+    h = hypot(primal(x), primal(y))
+    function hypot_pb!!(dh::P)
+        return NoRData(), dh * (primal(x) / h), dh * (primal(y) / h)
+    end
+    return CoDual(h, NoFData()), hypot_pb!!
+end
+
+@is_primitive MinimalCtx Tuple{typeof(hypot),P,P,Vararg} where {P<:IEEEFloat}
```

Comment:
```suggestion
@is_primitive MinimalCtx Tuple{typeof(hypot),P,P,Vararg{P}} where {P<:IEEEFloat}
```
It's possible to just be slightly more specific here.